virtual_rainforest/core/grid.py

```diff
@@ -158,45 +162,19 @@ def make_hex_grid(
     prototype = scale(prototype, xfact=scale_factor, yfact=scale_factor, origin=(0, 0))
     prototype = translate(prototype, xoff=xoff, yoff=yoff)
 
-    # Tile the prototypes to create the grid
-    cell_list = [None] * (cell_nx * cell_ny)
-
-    # Loop over columns and rows of cells, incrementing the base coordinates
-    for y_idx in range(cell_ny):
-        for x_idx in range(cell_nx):
-            # Define the cell id as integer count starting lower left by row
-            cell_id = x_idx + (y_idx) * cell_nx
-            # Store shifted geometry under cell id
-            cell_list[cell_id] = translate(
-                prototype,
-                xoff=2 * apothem * x_idx + apothem * (y_idx % 2),
-                yoff=1.5 * side_length * y_idx,
-            )
+    # Get the cell ids and centres of the cells
+    idx_y, idx_x = np.indices((cell_ny, cell_nx))
+    cell_ids = idx_x + idx_y * cell_nx
+    cell_x = 2 * apothem * idx_x + apothem * (idx_y % 2)
```

Comment:
Where is apothem coming from?

Reply:
I think this is GH being unhelpful - it hides chunks of code that are unchanged and the definition of apothem is in there. You can unpack the blue shaded lines from the left hand margin.